src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/ToModelTransformers.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.hmpps.hmppshdcapi.licences
+
+import uk.gov.justice.digital.hmpps.hmppshdcapi.model.HdcLicence as ModelHdcLicence
+
+/*
+** Functions which transform JPA entity objects into their API model equivalents.
+** Mostly pass-thru but some translations, so useful to keep the database objects separate from API objects.
+*/
+
+fun transformToHdcLicence(
```

Comment:
**suggestion:(non-blocking)** we could remove this and inline.

We know this pattern does not scale

Reply:
Happy to move this inline in the `LicenceService`.